app/javascript/angular/code/controllers/_fixed_sessions_map_ctrl.js

```diff
@@ -50,7 +51,8 @@ export const FixedSessionsMapCtrl = (
         mid: 35,
         high: 55,
         highest: 150
-      }
+      },
+      customTheme: false
```

Comment:
Is `theme` the word Michael uses for this feature?

Reply:
yes